db/migrate/20210908152154_create_receipt_details.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.references :receipt, null: false, foreign_key: true
       t.references :room, null: false, foreign_key: true
       t.datetime :from_time
-      t.datetime :to_time
+      t.datetime :end_time
```

Comment:
chỗ này rubocop nó không báo lỗi `Do not use `to_time` on Date objects` nên em đổi tên nha chị